test/sqllogictest/arithmetic.slt

```diff
@@ -498,6 +498,79 @@ SELECT round(CAST (5.0 AS float), 3.0)
 query error Cannot call function round\(double precision, double precision\): arguments cannot be implicitly cast to any implementation's parameters; try providing explicit casts
 SELECT round(CAST (5.0 AS float), CAST (3.0 AS float))
 
+query R
+SELECT trunc(CAST (1.5678 AS float))
+----
+1
+
+query R
+SELECT trunc(CAST (-1.4678 AS float))
+----
+-1
+
+query R
+SELECT trunc(CAST (1.5678 AS double precision))
+----
+1
+
+query R
+SELECT trunc(CAST (-1.4678 AS double precision))
+----
+-1
+
+query R
+SELECT trunc(1.5678)
+----
+1
+
+query R
+SELECT trunc(-1.5678)
+----
+-1
+
+query RRRRR
+SELECT trunc(1.1), trunc(1.111), trunc(100.1), trunc(100.11), trunc(-4.1)
+----
+1  1  100  100  -4
+
+# postgres converts ints to floats on trunc
+query R
+SELECT trunc(1)
+----
+1
+
+query R
+SELECT trunc(NULL)
+----
+NULL
+
+query R
+SELECT trunc(NULL::decimal)
+----
+NULL
+
+query R
+SELECT trunc((SELECT * FROM nums))
+----
+NULL
+
+statement ok
+DELETE FROM nums
+
+statement ok
+INSERT INTO nums VALUES (1.5678)
+
+query R
+SELECT trunc(1.5678)
+----
+1
+
+statement ok
+DROP TABLE nums
+
+query error
```

Comment:
this takes a third argument that is a partially line matching regex, something like: `query error No function matches`

Reply:
Good suggestion, thanks. Done